test/unit/sql/create-table.test.js

```diff
@@ -27,6 +27,31 @@ describe(Support.getTestDialectTeaser('SQL'), function() {
         });
       });
     });
+    if(Support.getTestDialect() === 'postgres') {
```

Comment:
`current.dialect.name === 'postgres'` is the convention for unit tests (we're moving to that generally)